templates/tonic-notifications-deployment.yaml

```diff
@@ -1,3 +1,7 @@
+{{- /* support values file w/o resource definition */}}
+{{- $resources := coalesce
+  ((((.Values).tonicai).notifications).resources)
+  (dict "limits" (dict "memory" "1Gi") "requests" (dict "memory" "512Mi" "ephemeral-storage" "1Gi")) -}}
```

Comment:
Parameterized resources were moved under the `tonicai.$service` keys. This is how the defaults are implemented. See similar things in the other templates below.